.circleci/config.yml

```diff
@@ -1064,8 +1064,8 @@ jobs:
 
   fpp-verify:
     circleci_ip_ranges: true
-    docker:
-      - image: cimg/go:1.21
+    machine: true
+    resource_class: swc/ax101
```

Comment:
Do we have a clear policy about when to use `swc/ax101`?

Reply:
Not clear. Currently, if the job costs more than 10 minutes or requires large `xlarge` resource class, I'll use `swc/ax101` to reduce time and costs.  @qzhodl

Reply:
We may consider transitioning all jobs to ax101 in the long term to minimize the credit costs associated with CircleCI